src/decorators/dispatch.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  NgRedux,
+} from '../components/ng-redux';
+
+
+export function dispatch(options?: any) {
```

Comment:
Any plans to use `options` anywhere? Not being used anywhere (maybe required by TypeScript, not too familiar with the language yet).